data/tabular/sarscov2_vitro_touret/transform.py

```diff
@@ -56,6 +56,9 @@ def get_and_transform_data():
                     {"adjective": "active against the Corona virus"},
                     {"adjective": "active against SARSCoV2"},
                     {"adjective": "active against COVID19"},
+                    {"gerund": "fighting the Corona virus"},
```

Comment:
"fighting" sounds a little bit like slang here, maybe there is a more formal word?

Reply:
countering?